tljh_repo2docker/__version__.py

```diff
@@ -0,0 +1 @@
+__version__ = "1.0.0"
```

Comment:
Maybe we won't be able to release `1.0.0` with the releaser directly, because the version will need to be bumped?

Reply:
:+1:

Reply:
we are good to go?

Reply:
`APP_ID` should be in variable section?

Reply:
> `APP_ID` should be in the variable section?

Ah good catch, thanks!

Updated:

![image](https://github.com/plasmabio/tljh-repo2docker/assets/591645/e6b2b31b-3158-4d1e-80fb-9097d4fb286e)